CONTRIBUTOR-LADDER.md

```diff
@@ -0,0 +1,134 @@
+# Contributor Ladder
+
+* [Contributor Ladder](#contributor-ladder)
+    * [Community Contributor](#community-contributor)
+    * [Organization Member](#organization-member)
+    * [Reviewer](#reviewer)
+    * [Committer](#committer)
+
+## Contributor Ladder
+
+Hello! We are excited that you want to learn more about our project contributor ladder! This contributor ladder outlines the different contributor roles within the Cilium project, along with the responsibilities and privileges that come with them. Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows. Our contributors are happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into lists of three types of things. "Requirements" are things that a contributor is expected to do at that level. "Qualifications" are qualifications a person needs to meet to be at that level (may be one or any/all of them), and "Privileges" are things contributors on that level are entitled to.
+
+As the Cilium project grows, the current roles may be broken out into new roles and/or teams and roles may no longer be needed. 
+
+The final judgment on whether an individual fulfills the criteria for a role is up to the Cilium Committers. After six months of inactivity, any contributor can be removed from their position(s).
```

Comment:
I wonder if it's also worth linking the revocation policy and clarifying that it applies to all roles as well?

https://docs.cilium.io/en/v1.13/community/governance/commit_access/#revoking-commit-access

The above is primarily written around committer access, but this could apply equally to other roles, for instance if a contributor can run CI and begins to abuse it, or if they are misusing access to community social accounts.

Reply:
good point, I've added "or behavior detrimental to the future of the project"